trac-env/htdocs/css/trachacks.scss

```diff
@@ -274,15 +274,17 @@ div[role="main"]{
       }
     }
 
-    .inlinebuttons input[type="submit"] {
+    .inlinebuttons input[type="submit"],
+    button.trac-revert {
       background: #ddd;
       color: #444;
       font-size: 1.1rem;
-    }
+      height: auto;
```

Comment:
what does `height: auto` do? (I thought that was the default, so I'm not sure why it needs to be declared explicitly).